### ISSUE TITLE
Training multiband melgan on predicted mel

### ISSUE THREAD

**Issue by cnamuangtoun** [TensorSpeech/TensorFlowTTS#755]
Hi, I'm just curious if training the multiband melgan with predicted mel spectogram from fastspeech2 would imrpove the quality of the waveform output from the vocoder instead of using the ground truth mel spectogram straight from the dataset

**Comment by xianglan010** [comment]
I try to use the mel spec generated from pertained tacotorn2 to train the MB-melgan, but the loss not convergence, have you tried it and how about ur result
